src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedByDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
+
+  const splittedByDoubleDot = splittedByDotComa
+    .map((item) => item.split(':'))
+    .map((SplittedCssProperty) =>
+      // eslint-disable-next-line prettier/prettier
+      SplittedCssProperty.map((propertyPart) => propertyPart.trim()));
+
+  const cssProperiesObject = splittedByDoubleDot.reduce((acc, [key, value]) => {
+    acc[key] = value;
+
+    return acc;
```

Comment:
Using 'reduce' directly on 'splittedByDoubleDot' is the correct approach, but consider using object spread syntax for updating the accumulator object. This can lead to cleaner and more immutable-style code.

Reply:
fixed